fastlane/spec/fixtures/plugins/GemfileWithDeps/dependency_with_plugins/dependency_with_plugins.gemspec

```
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+Gem::Specification.new do |spec|
+  spec.name        = 'dependency_with_plugins'
+  spec.version     = '1.0.0'
+  spec.authors     = ["Fastlane team"]
```

```suggestion
  spec.authors     = ["fastlane team"]
```
_But Roger, even in fixtures??_  😝